aws/resource_aws_spot_fleet_request_test.go

```diff
@@ -1504,8 +1466,8 @@ resource "aws_launch_template" "test" {
 }
 
 resource "aws_spot_fleet_request" "test" {
-  iam_fleet_role                      = "${aws_iam_role.test-role.arn}"
-  spot_price                          = "0.05"
+  iam_fleet_role                      = "${aws_iam_role.test.arn}"
+  spot_price                          = "0.005"
```

Comment:
Please leave the prices as is until we move to a pricing data source.

Reply:
Will do; Got overeager during the rebase.

Reply:
Done.